eZ/Bundle/EzPublishCoreBundle/Command/NormalizeImagesPathsCommand.php

```diff
@@ -153,6 +156,22 @@ private function updateImagePath(int $fieldId, string $oldPath, string $newPath)
         $newFilename = $newPathInfo['basename'];
         $newBaseName = $newPathInfo['filename'];
 
+        // Checking if a file exists physically
+        $oldBinaryFile = $this->ioService->loadBinaryFileByUri(\DIRECTORY_SEPARATOR . $oldPath);
+        if ($oldBinaryFile instanceof MissingBinaryFile) {
+            $io->warning(sprintf('Skipping file %s as it doesn\'t exists physically.', $oldPath));
+
+            return;
+        }
+
+        try {
+            $inputStream = $this->ioService->getFileInputStream($oldBinaryFile);
+        } catch (BinaryFileNotFoundException $e) {
+            $io->warning(sprintf('Skipping file %s as it doesn\'t exists physically.', $oldPath));
+
+            return;
+        }
```

Comment:
This is quite redundant. What is going to happen if you pass an instance of `MissingBinaryFile` to `IOServiceInterface::getFileInputStream`?